docs/core/deploying/reducing-dependencies.md

```diff
@@ -12,15 +12,15 @@ ms.devlang: dotnet
 ms.assetid: 916251e3-87f9-4eee-81ec-94076215e6fa
 ---
 
-# How to trim your package dependencies
+# Reducing Package Dependencies with project.json
 
-This article covers what you need to know about trimming your package dependencies to use only the packages you need for .NET Core 1.0.
+This article covers what you need to know about reducing your package dependencies when authoring `project.json` libraries. By the end of this article, you will learn how to compose your library such that it only uses the dependencies it needs. 
 
-## What trimming means
+## Why it's Important
 
 .NET Core 1.0 introduces the [.NET Standard Library metapackage](https://www.nuget.org/packages/NETStandard.Library/1.6.0), which is a NuGet package composed of other packages.  It is curated to be appropriate for developing libraries.  However, there's a good chance that your library won't use every single package it contains.  When authoring a library and distributing it over NuGet, it's a best practice to "trim" your dependencies down to only the packages you actually use.  This results in a smaller overall footprint for NuGet packages.
```

Comment:
Would be appropriate to link here: https://github.com/dotnet/core-docs/blob/master/docs/core/packages.md